sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -94,18 +93,19 @@ public virtual Response UpdateThread(string topic, CancellationToken cancellatio
 
         #region Message Operations
         /// <summary> Sends a message to a thread asynchronously. </summary>
-        /// <param name="content"> Message content. </param>
-        /// <param name="priority"> Message priority. </param>
-        /// <param name="senderDisplayName"> The display name of the message sender. This property is used to populate sender name for push notifications. </param>
+        /// <param name="content"> Chat message content. </param>
+        /// <param name="type"> The chat message type. </param>
+        /// <param name="senderDisplayName"> The display name of the chat message sender. This property is used to populate sender name for push notifications. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual async Task<Response<SendChatMessageResult>> SendMessageAsync(string content, ChatMessagePriority? priority = null, string senderDisplayName = null!, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<string>> SendMessageAsync(string content, ChatMessageType? type = null, string? senderDisplayName = null, CancellationToken cancellationToken = default)
```

Comment:
Question to think about, between display name and type which one do we think is more likely to be set? If display name is more important, we should probably swap the order.